rest-service/manager_rest/test/endpoints/test_app.py

```diff
@@ -29,3 +32,56 @@ def test_get_root_404(self):
         """
         resp = self.app.get('/')
         self.assertEqual(404, resp.status_code)
+
+    def test_does_not_init_tracer(self):
+        # Dummy call to invoke the tracer initialization if enabled.
+        self.app.get('/')
+        self.jaeger_mock_config.assert_not_called()
+
+
+@attr(client_min_version=base_test.LATEST_API_VERSION,
+      client_max_version=base_test.LATEST_API_VERSION,
+      setup_config_enable_tracing=True,
```

Comment:
use test fixture of pytest, not this.
attr is for running a test with a filter specified here